exporter/pwd_file_test.go

```diff
@@ -0,0 +1,16 @@
+package exporter
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestLoadPwdFile(t *testing.T) {
```

Comment:
The test still needs a little polishing. See what the other tests do in terms of using t.Fatalf() if expected data wasn't found in the result of `LoadPwdFile()`